.github/workflows/engine.yml

```
@@ -90,3 +114,50 @@ jobs:
+        uses: actions/download-artifact@v4
+        with:
+          name: echo-inferlet
+          path: ./bin
+      
+      - name: Make binaries executable
+        run: |
+          chmod +x ./bin/pie
+          chmod +x ./bin/pie-cli
+
+      - name: Add binaries to PATH
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:1:22: Double quote to prevent globbing and word splitting

(shellcheck)